orcestra/postprocess/level1.py

```diff
@@ -148,6 +149,45 @@ def _filter_spikes(ds, threshold=5, window=1200):
     return xr.where(abs(diff) < threshold, ds, interpolated)
 
 
+def _filter_land(ds, mask, lat, lon, offset=pd.Timedelta("7s")):
+    """
+    Filters out data that was collected over land.
+    Removes data by offset earlier than the the time the plane flies over land
+    to avoid including land measurements due to tilt of the plane or mask inaccuracies.
+
+    Parameters
+    ----------
+    ds : xr.Dataset
+        Dataset to filter.
+    mask : xr.DataArray
+        Mask of land and sea. 1 for sea, 0 for land.
+    lat : xr.DataArray
+        Latitudes of the path.
+    lon : xr.DataArray
+        Longitudes of the path.
```

Comment:
Would it be safe to assume that the input dataset is well-formatted and thus has a `lat` and `lon` variable? This way one could simplify the function signature a lot